modules/ROOT/pages/cloudhub-deploy-options.adoc

```diff
@@ -3,21 +3,39 @@ ifndef::env-site,env-github[]
+settings of Access Manager's business group. Subsequent deployments are deployed to the same region as the first deployment.
 
 To access the default deployment settings for your organization, navigate to *Access Management > Partner Manager*.
 
 You can configure the following Partner Manager Settings for CloudHub deployment.
 
-* Mule runtime version
+* Runtime version
+
+The default and recommended option is `Use latest version`, which deploys Partner Manager new message flows on the latest Mule runtime version.
+If you want the applications deployed to a specific older version, choose the appropriate version from the menu.
+
+* CloudHub region
+
+The default option is retrieved from the business group’s default region setting for Runtime Manager in Access Manager.
+If you want the B2B integration flows from Partner Manager to be deployed to a different region,
+select it from the drop-down menu prior to deploying your first message flow.
+Subsequent deployments get deployed to the same region as the first deployment.
+To change the default region moving forward, you must undeploy and redeploy all the previously deployed message flows.
+For more information refer to xref:configure-message-flows.adoc[Configure Message Flows].
+
+* Anypoint MQ region
 
-The default and recommended option is ‘Use latest version’, which deploys Partner Manager new message flows on the latest Mule runtime version. If you want the applications deployed to a specific older version, choose the appropriate version from the menu.
+This is automatically chosen based on the CloudHub region and cannot be changed.
 
 * Automatically restart app when not responding
 
-If your system allows, you can set this to restart Partner Manager if it doesn’t respond. For more information, refer to Automatic Restart.
+If your system allows, you can set this to restart Partner Manager if it does not respond.
+For more information, refer to https://docs.mulesoft.com/runtime-manager/worker-monitoring#enable-automatic-restarts[Enable Automatic Restarts].
 
 * vCore Settings
 
-These settings are configurable at the environment type level (such as Sandbox or Production). The `Document Processing` and `API Replication Services` are core components of the Partner Manager workflow, requiring a minimum of 1 vCore each. Each unique trading partner and internal endpoints get deployed to isolated workers - requiring a minimum of 0.1 vCore each. If you require assistance with sizing, please contact your Account representative.
+Configure these at the Sandbox or Production environment level.
+The `Document Processing` and `API Replication` Services are core components of the Partner Manager workflow,
+requiring a minimum of 1 vCore each. Each unique trading partner and internal endpoints get deployed to isolated workers - requiring a minimum of 0.1 vCore each.
+If you require assistance with sizing, please contact your Account representative.
```

Comment:
Are we mentioning account reps anymore, or has that changed to just Support, a la Salesforce?
```suggestion
If you require assistance with sizing, contact your account representative.
```